package-res/reportviewer/reportviewer-prompt.js

```diff
@@ -94,8 +94,6 @@ define(['common-ui/util/util', 'pentaho/common/Messages', "dijit/registry", 'com
             if (initFlag) {
               this._createPromptPanelFetchCallback(paramDefn);
```

Comment:
@pentaho/orlandocalisbon could we remove the need for this initFlag, using something like [_.once](http://underscorejs.org/#once) ?